static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -151,13 +107,30 @@ const modeText = {
 
 export default function ResourcePickerDialog(props: Props): JSX.Element {
   const { mode, isOpen, closeDialog, insertEmbed, contentNames } = props
+  const website = useWebsite()
+  const definedCategories = useMemo(() => {
+    return website.starter?.config?.collections
+      .filter(entry => entry.category === "Content")
+      .flatMap(elt => [elt.name, elt.label])
+  }, [website.starter?.config?.collections])
   const tabs = useMemo(
     () =>
       contentNames
-        .flatMap(name => TABS[name])
+        .flatMap(name => {
+          if (name && name === "resource") {
+            return [documentTab, videoTab, imageTab, otherFilesTab]
+          } else if (
+            definedCategories &&
+            definedCategories?.indexOf(name) !== -1
+          ) {
+            const idx = definedCategories?.indexOf(name)
+            return [toTabSettings(definedCategories?.slice(idx, idx + 2))]
```

Comment:
With the above suggestion, you should be able to just lookup the category by name: `definedCategories[name]`.

Reply:
Updated in https://github.com/mitodl/ocw-studio/pull/1448/commits/b46ad80309a35e46f7487f9c547179a8bcbe6fe2.